### ISSUE TITLE
Basic tutorial on how to use it

### ISSUE THREAD

**Issue by SamDecrock** [kubemove/kubemove#36]
Can you please write up a basic tutorial with examples on how to use KubeMove?

It seems like an interesting module which I was about to write myself ... 

Thanks!

**Comment by mynktl** [comment]
Hi @SamDecrock 

Thanks for showing interest in Kubemove.
There is one [user guide](https://github.com/kubemove/elasticsearch-plugin/blob/master/docs/user-guide.md), with detailed steps and example YAML, to setup the Kubemove framework and elasticsearch plugin.
Please let me know if you need any more help.

Thanks!

**Comment by jeffreybrowning** [comment]
@mynktl similarly to @SamDecrock, it would be great to get a manual on basic, intermediate and advanced use cases. The elasticsearch example I would categorize as at least intermediate.

Connecting two clusters and moving a single deployment might be the best and simplest tutorial?

**Comment by SamDecrock** [comment]
Yes, moving a single deployment with one or 2 pvc's would be nice.
Because in essence, that's the thing I want to solve: move the content of the pv(c)'s.

**Comment by SamDecrock** [comment]
If I understand your project correctly, you're actually moving on an application level, right?
KubeMove uses plugins to do this and you created one plugin for elasticsearch already? Are there more plugins, for mysql for example?

**Comment by jeffreybrowning** [comment]
@mynktl for 👀 . 

Is this a project you are still working on? If you're busy, can you point us to another contributor that could take a stab at a bare-bones example? 

Once we have that, I think myself and others could help with writing more complex documentation.

**Comment by jeffreybrowning** [comment]
popping in some more @'s to get the conversation going:

@umamukkara @ashishranjan738

**Comment by mynktl** [comment]
Hi @jeffreybrowning @SamDecrock 
Sorry for the delay in reply.

> If I understand your project correctly, you're actually moving on an application level, right?
> KubeMove uses plugins to do this and you created one plugin for elasticsearch already? Are there more plugins, for mysql for example?

Yes. Kubemove framework uses a plugin to move the application data to the remote cluster. As of now, we have only elasticsearch app plugin which uses the s3 base repository to transfer the data.



> @mynktl for eyes .
> 
> Is this a project you are still working on? If you're busy, can you point us to another contributor that could take a stab at a bare-bones example?
> 
> Once we have that, I think myself and others could help with writing more complex documentation.

As of now, I am not working on this project. I've prepared one [document](https://docs.google.com/document/d/1HlZIOU1xgaRnJZxHD3d6hAGBjEciv6z63SzNw1PH5dw) on installing/configuring kubemove-framework/elasticsearch-plugin. 

You can also refer to the following links to install/configure kubemove/elasticsearch app.
https://github.com/kubemove/elasticsearch-plugin/blob/master/docs/user-guide.md
https://github.com/kubemove/elasticsearch-plugin/blob/master/docs/user-guide.md

Let me know if you need any help. 

Thanks!

**Comment by jeffreybrowning** [comment]
@mynktl Thanks for the resources -- I think you may need to open up permissions on the document you linked -- it says that I do not have access to view.

Is there another contributor to this project that we could work with to onboard to contributing? I looked at the meeting notes for the recurring Monday meetings, but they do not seem to have been updated since April of last year.

I would love to help out and keep an open-source project to move resources b/w clusters, rather than letting this fall to the wayside in favor of Anthos or other platform specific tools. That would be a real bummer.

**Comment by mynktl** [comment]
Hi @jeffreybrowning 
I've changed access permission. Can you access it now? link is https://docs.google.com/document/d/1HlZIOU1xgaRnJZxHD3d6hAGBjEciv6z63SzNw1PH5dw/edit?usp=sharing

**Comment by jeffreybrowning** [comment]
@mynktl yup! I can access it now. Thank you. 

Just to circle back to my other question though. I don't want to bother you, but also don't want this thread to die.

Are there active maintainers on this project? If so, can you point me to them?

**Comment by mynktl** [comment]
Hi @jeffreybrowning 
As of now, there are no any active maintainers in this project.

**Comment by mynktl** [comment]
> @mynktl yup! I can access it now. Thank you.
> 
> Just to circle back to my other question though. I don't want to bother you, but also don't want this thread to die.
> 
> Are there active maintainers on this project? If so, can you point me to them?

Hi @jeffreybrowning 
As of now, there are no active maintainers in this project.

**Comment by jeffreybrowning** [comment]
Sad news indeed @mynktl . But thanks for the update. I'll try my hand at getting this up and running and writing a tutorial myself. If and when that happens, would assigning the PR to you be correct?

**Comment by mynktl** [comment]
yes @jeffreybrowning.
